.clang-tidy

```diff
@@ -9,6 +9,7 @@ Checks:
   -modernize-use-trailing-return-type,
   -modernize-concat-nested-namespaces,
   -modernize-use-default-member-init,
+  -modernize-macro-to-enum,
```

Comment:
That's because there are such macros in the generated Grammar.cpp _before_ we can generate the NOLINTBEGIN.